src/sentry/performance_issues/base.py

```diff
@@ -379,10 +379,12 @@ def fingerprint_http_spans(spans: list[Span]) -> str:
     """
     Fingerprints http spans based on their paramaterized paths, assumes all spans are http spans
```

Comment:
im going to clean up `base.py` this week so will be able to remove this import with that